node_modules/oae-core/editprofile/editprofile.html

```diff
@@ -1,70 +1,134 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/editprofile.css" />
+
 <!-- MODAL -->
 <div id="editprofile-modal" class="modal fade" tabindex="-1" aria-labelledby="editprofile-modal-title" aria-hidden="true">
     <div class="modal-dialog">
         <div class="modal-content">
-            <div class="modal-header">
-                <button type="button" class="close" data-dismiss="modal" aria-label="__MSG__CLOSE__">
-                    <span aria-hidden="true">&#215;</span>
-                </button>
-                <h3 id="editprofile-modal-title">__MSG__EDIT_DETAILS__</h3>
+            <div id="editprofile-panel-main-container"><!-- --></div>
+            <div id="editprofile-panel-email-container" class="hide"><!-- --></div>
             </div>
-            <form id="editprofile-form" role="form">
-                <div class="modal-body"><!-- --></div>
-                <div class="modal-footer">
-                    <button type="button" class="btn btn-link" data-dismiss="modal">__MSG__CANCEL__</button>
-                    <button type="submit" class="btn btn-primary" disabled="true">__MSG__SAVE__</button>
-                </div>
-            </form>
         </div>
     </div>
 </div>
 
-<div id="editprofile-template"><!--
-    <div class="form-group">
-        <label for="editprofile-name" class="control-label"><h4>__MSG__NAME__</h4></label>
-        <input type="text" id="editprofile-name" class="form-control required maxlength-short displayname" name="editprofile-name" placeholder="__MSG__ENTER_A_NAME__" value="${oae.data.me.displayName|encodeForHTMLAttribute}"/>
+<div id="editprofile-panel-email-template"><!--
+    <div class="modal-header">
+        {if isValidProfile}
+            <button type="button" class="close" data-dismiss="modal" aria-label="__MSG__CLOSE__">
+                <span aria-hidden="true">&#215;</span>
+            </button>
+        {/if}
+        <h3 id="editprofile-modal-title">__MSG__PROFILE_SUCCESSFULLY_UPDATED__</h3>
+    </div>
+    <div id="editprofile-email-container" class="modal-body">
+        <div class="alert alert-warning">
+            __MSG__PROFILE_UPDATED_VERIFICATION_INSTRUCTION__
```

Comment:
We need 2 different instructions here. One that is used when the first unverified email is set and the user can't use the system yet. An other when a new unverified email is set but there already was a verified one, so the user can continue using the system.